src/img/worker.py

```diff
@@ -351,7 +352,7 @@ def runmic(self, ssh=False, job_args=None):
             mic_comm.append('--config=%s' % job_args.ksfile_name)
         elif self.ict == "mic":
             mic_comm.append('%s' % job_args.image_type)
-            mic_comm.append('"%s"' % job_args.ksfile_name)
+            mic_comm.append('%s' % job_args.ksfile_name)
```

Comment:
Notice how mic_comm is passed either to ssh() or run() below, based on some condition. The extra quotes fixes the original ssh() invocation but otoh would break the run() invocation.